Packs/AWS-Enrichment-Remediation/Scripts/AWSRecreateSG/AWSRecreateSG.yml

```diff
@@ -17,6 +17,8 @@ args:
   required: true
 - description: Name of an AWS role to assume (should be the same for all organizations).
   name: assume_role
+- description: Region where EC2 instance is present.
+  name: region
```

Comment:
I would prefer not to do this because:

- AWS [adds regions](https://aws.amazon.com/about-aws/global-infrastructure/#:~:text=AWS%20Global%20Infrastructure%20Map,%2C%20New%20Zealand%2C%20and%20Thailand.) and we would have have to continue to add them
- This is a optional argument

Reply:
This is true for instance parameters (you can only pick an item from the list and nothing else), but for command parameters you can either choose from the list (which is narrowed down as you type and allows you to auto-complete) or enter custom text. You can update the list if you'd like.

The field being optional is unrelated to the predefined option, it will remain optional.

Reply:
Okay, you convinced me
[add predefined regions](https://github.com/demisto/content/pull/29840/commits/877b131ce9d33219dd9d15c6265d0ddf8182c7ce)